### ISSUE TITLE
Remove "The plugin kirki/kirki.php has been deactivated due to an error: Plugin file does not exist."

### ISSUE THREAD

**Issue by vlthemes** [themeum/kirki#1448]
### Issue description:

Hi Ari.

How can I remove this error?
 "The plugin kirki/kirki.php has been deactivated due to an error: Plugin file does not exist."

### Version used:
dev

**Comment by danielortiz** [comment]
judging by the lines of code below, it seems that this error only happens if the plugin is not  correctly installed on the right folder, or maybe there's missing files on its installation
```
function validate_plugin($plugin) {
	if ( validate_file($plugin) )
		return new WP_Error('plugin_invalid', __('Invalid plugin path.'));
	if ( ! file_exists(WP_PLUGIN_DIR . '/' . $plugin) )
		return new WP_Error('plugin_not_found', __('Plugin file does not exist.'));

	$installed_plugins = get_plugins();
	if ( ! isset($installed_plugins[$plugin]) )
		return new WP_Error('no_plugin_header', __('The plugin does not have a valid header.'));
	return 0;
}
```
try reinstalling it

**EDIT:** 
Weird enough, when I visited the plugins settings page on my wp admin, I got the same message for all my plugins 🤔. So I'm not sure if it's a problem directly related to Kirki. Anyway, simply reactivating them did the trick

**Comment by vlthemes** [comment]
I don't use plugin.

**Comment by AdsonCicilioti** [comment]
follow this https://aristath.github.io/kirki/docs/advanced/integration.html ?

**Comment by vlthemes** [comment]
It works on 2.x but not dev :(

**Comment by aristath** [comment]
@vlthemes can you please tell me how to replicate this issue?
Do you have Kirki installed as a plugin or embedded in a theme?
If in a theme do you use any special code to embed it?
Is it also installed as a plugin?
Where do you get that message? Is it in the plugins screen? 
Multisite or singlesite? 
If on multisite do you get this on the network plugins page? Or when viewing a single site on the multisite?

I'm unable to replicate this issue so please provide specific steps and as many details as possible so that I may replicate and fix it.

**Comment by vlthemes** [comment]
This error has disappeared for no apparent reason. Embedded in theme. At the first installation, it appears again and disappears.

I recently sent you my theme.

**Comment by aristath** [comment]
Fixed, will be included in 3.0.9.
